contracts/migrations/2_deploy_v0.js

```diff
@@ -1,7 +1,6 @@
 // Load zos scripts and truffle wrapper function
 const { scripts, ConfigManager } = require('@openzeppelin/cli');
```

Comment:
Check out the changes here @moose-code

Reply:
Have done! Very interesting thanks! Interested to talk about the subtleties of this file change and why the previous code failed